pkg/util/k8sutil/pod_util.go

```diff
@@ -86,7 +86,7 @@ func PodWithAntiAffinity(pod *v1.Pod, clusterName string) *v1.Pod {
 				{
 					LabelSelector: &metav1.LabelSelector{
 						MatchLabels: map[string]string{
-							"etcd_cluster": clusterName,
+							"app": "etcd",
```

Comment:
i think this is only required for self hosted cluster. for non self hosted, clusterName is good enough.